app/script/components/ephemeralTimer.coffee

```diff
@@ -21,54 +21,32 @@ z.components ?= {}
 
 class z.components.EphemeralTimer
   constructor: (params) ->
-    scheduled_time = new Date(params.expires()).getTime()
-    starting_time = Date.now()
-    timer = scheduled_time - starting_time
-    update_interval = 1000 / 10
+    message_et = params.message
+    ephemeral_expires = new Date(message_et.ephemeral_expires()).getTime()
+    ephemeral_started = new Date(message_et.ephemeral_started()).getTime()
+
```

Comment:
with introducing this value there might be the possibility that we load a message from the db that does not have a started time. This would result in `new Date('').getTime() = NaN`